"Contact Us" Confirmation Page

BattleDakota opened the pull request:
Creation of confirmation page and added redirect upon submission of contact form while retaining functionality

BattleDakota committed:
"Contact Us" Confirmation Page

Creation of confirmation page and added redirect while retaining functionality

QJMTech reviewed (commented):
Awesome work! Not going to merge this just yet, as I actually have the confirmation being served only after the POST request gets through to the flask app and the email is sent, whereas this would have it fire right after clicking the button (meaning that it may send them to a confirmation page even if their email hasn't been sent), but I will find a way to make flask serve this page after the request is processed, so it definitely will be used!!!

QJMTech committed:
Update contact.html

Changing button to function as before so Flask can serve webpage instead of button redirecting to it
